### PR TITLE
OT123-354 | Testing members endpoints

### PR DESCRIPTION
**Pull request by DarioMJ48**
[OT123-324](https://alkemy-labs.atlassian.net/jira/software/c/projects/OT123/boards/190?modal=detail&selectedIssue=OT123-354&assignee=61e6dca908c1f7006a1c28d7)

GET, POST, PUT & DELETE tests all working.
Minor typos fixed at members route & controller.

members

### CODE REVIEW

File: controllers/members.js [hunk 1]
```diff
@@ -74,9 +68,9 @@ exports.editMember = async (req, res) => {
   member.image = image
   await member
     .save()
-    .then((updatedMember) => res.status(201).json({
+    .then((updatedMember) => res.status(200).json({
```
Comment (dvelezroman) [inline controllers/members.js:71]: 201 es el codigo para actualizar... 
[https://www.restapitutorial.com/httpstatuscodes.html](https://www.restapitutorial.com/httpstatuscodes.html)

File: controllers/members.js [hunk 2]
```diff
@@ -96,14 +90,14 @@ exports.deleteMember = async (req, res) => {
...
     }
     await member
       .destroy()
     return res.status(200).json({
       ok: true,
-      msg: 'member was deleted',
+      msg: 'Member was deleted.',
     })
   } catch (err) {
     res.status(400).json({
```
Comment (dvelezroman) [inline controllers/members.js:103]: aqui seria error 500

Comment (dvelezroman) [reply controllers/members.js:103]: https://www.restapitutorial.com/httpstatuscodes.html

**Review by dvelezroman — APPROVED** [review]
arreglar esos dos status code.. y revsar esto para tener un mejor contextos de los codigos de REST API 
https://www.restapitutorial.com/httpstatuscodes.html

File: controllers/members.js (hunk 1 above)
Comment (DarioMJ48) [reply controllers/members.js:71]: Dale, aunque yo no hice ese controlador, solo lo modifiqué.

**Review by patriciodfernandez — APPROVED** [review]

**Pull request merged**